samples/rust/rust_rofs.rs

```diff
@@ -17,17 +17,3 @@ struct RoFs;
 impl fs::FileSystem for RoFs {
     const NAME: &'static CStr = c_str!("rust-fs");
 }
-
-#[pin_data]
-struct RoFsModule {
-    #[pin]
-    fs_reg: fs::Registration,
-}
-
-impl kernel::InPlaceModule for RoFsModule {
-    fn init(module: &'static ThisModule) -> impl PinInit<Self, Error> {
-        try_pin_init!(Self {
-            fs_reg <- fs::Registration::new::<RoFs>(module),
-        })
-    }
-}
```

Comment:
I do not think that this makes it easier to understand/review, so I would move the sample module after the `module_fs` patch.

Reply:
I was trying to showcase that the macro simplified things, but indeed people may be caught up on this step. I'll remove it.